components/IllustratedListBlock/src/stories/bem.stories.mdx

```diff
@@ -0,0 +1,91 @@
+import { Canvas, Description, Meta, Story } from "@storybook/addon-docs";
+import { DesignTokenDocBlock } from "storybook-design-token/dist/doc-blocks";
+import "../index.scss";
+import pkg from "../../package.json";
+import readme from "../../README.md";
+
+<Meta
+  title="CSS/Data Display/Illustrated List Block"
+  parameters={{
+    componentSubtitle: `${pkg.name} - ${pkg.version}`,
+    status: {
+      type: "WORK IN PROGRESS",
+    },
+  }}
+/>
+
+<Description markdown={readme} />
+
+## Design Tokens
+
+<DesignTokenDocBlock categoryName="Illustrated List Block" viewType="table" />
+
+## Stories
+
+### Default
+
+<Canvas>
+  {/* eslint-disable react/no-unknown-property */}
+  <Story name="Default">
+    <div class="denhaag-illustrated-list-block">
+      <div class="denhaag-illustrated-list-block__content">
+        <h3 class="utrecht-heading-4">Al uw identiteitskaarten en paspoorten</h3>
+        <p class="utrecht-paragraph">Heeft u een identiteitskaart én een paspoort? Neem alles mee. </p>
+      </div>
+      <img
```

Comment:
Meer een suggestie dan een opmerking. Omdat we images lazy loaden (zoals het hoort!), is het wenselijk om een background animation te hebben? Bijv een skeleton loader pulse oid. Weet niet hoe GDH hierin staat. Voor de rest prima